.scripts/before_deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rm -rf vendor
+composer install --no-dev
```

Comment:
Case you want, you can verify if `composer` is installed

Reply:
When using php on travis composer will be installed automatically. If its not, then I want build to fail anyway. Also, I'm already running composer on `install: composer update`, so, if it isn't installed it will fail before.